README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_2048_game/)
+    - [DEMO LINK](https://NVlasenko.github.io/js_2048_game/)
```

Comment:
your demo is not deployed (there is only readme file)
please, make sure your app is deployed (ask in the chat if you need help) and write your link in the PR description